README.md

```diff
@@ -37,6 +37,7 @@
   - [Body processing](#body-processing)
     - [Response serialization](#response-serialization)
     - [Response deserialization serialization](#response-deserialization-serialization)
+- [Compression](#compression)
```

Comment:
Seems like the wrong level. Should be nested in Usage

Reply:
fixed